content/prerequisites/cloud9/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Use Cloud9 IDE"
+weight: 200
+---
+
+## What is Cloud9
+
+AWS Cloud9 is a cloud-based IDE that also offers a command-line terminal. It already comes with AWS CLI, Node.js, AWS CDK Toolkit, and Git installed by default. That's why we strongly recommend you to consider AWS Cloud9 to run this workshop. You can configure a cloud watch instance from aws console as well using aws cloud formation.
```

Comment:
```suggestion
AWS Cloud9 is a cloud-based IDE that lets you write, run, and debug your code with just a browser. It includes a code editor, debugger, and terminal. Cloud9 comes prepackaged with essential tools for running this workshop. Since your Cloud9 IDE is cloud-based, you can do labs from your office, home, or anywhere using an internet-connected machine. 
```